kvdb-memorydb/src/lib.rs

```diff
@@ -33,23 +33,24 @@ pub fn create(num_cols: u32) -> InMemory {
 	InMemory { columns: RwLock::new(cols) }
 }
 
+fn invalid_column(col: u32) -> io::Error {
+	io::Error::new(io::ErrorKind::Other, format!("No such column family: {:?}", col))
+}
+
 impl KeyValueDB for InMemory {
 	fn get(&self, col: u32, key: &[u8]) -> io::Result<Option<DBValue>> {
 		let columns = self.columns.read();
 		match columns.get(&col) {
-			None => Err(io::Error::new(io::ErrorKind::Other, format!("No such column family: {:?}", col))),
+			None => Err(invalid_column(col)),
```

Comment:
the kvdb-rocksdb implementation panics in `fn cf` on invalid column
should we do the same here?

Reply:
I would not panic, at least not on purpose.

Reply:
Pushed [2d9040d](https://github.com/paritytech/parity-common/pull/661/commits/2d9040d907223a804a4f28c1fd9b0714af93ee09), but it feels like extra abuse of `io::Error`. Ideally, we'd need to introduce our own `Error` type. But I'm don't with refactorings for today :)